service/policy/db/query.sql

```diff
@@ -108,12 +115,142 @@ SET
     uri = COALESCE(sqlc.narg('uri'), uri),
     public_key = COALESCE(sqlc.narg('public_key'), public_key),
     name = COALESCE(sqlc.narg('name'), name),
-    metadata = COALESCE(sqlc.narg('metadata'), metadata)
+    metadata = COALESCE(sqlc.narg('metadata'), metadata),
+    source_type = COALESCE(sqlc.narg('source_type'), source_type)
 WHERE id = $1;
 
 -- name: DeleteKeyAccessServer :execrows
 DELETE FROM key_access_servers WHERE id = $1;
 
+
+-----------------------------------------------------------------
+-- Key Access Server Keys
+------------------------------------------------------------------
+-- name: CreateKey :one
```

Comment:
Can you please lowercase all of these queries to align with the open issue we have to internalize them? https://github.com/opentdf/platform/issues/1652
```suggestion
-- name: createKey :one
```